packages/fxa-auth-server/bin/key_server.js

```diff
@@ -26,6 +26,7 @@ function run(config) {
   }
 
   const log = require('../lib/log')({ ...config.log, statsd });
+  require('../lib/oauth/logging')(log);
```

Comment:
oauth-server uses a different logging style and initialization strategy. This welds the two together until it's convenient to refactor.